### ISSUE TITLE
Brayden  Umbrella Ticket Completed Ticket-

### ISSUE THREAD

**Issue by BraydenBasinger** [noahahooja/p4schlimes#16]
Completed Ticket Code Here:
https://github.com/noahahooja/p4schlimes/commit/7acbe1f39f9202866d63fbb2f93df8cdb9fa7e60

I did this to see a stock chart in a website. It looks like an embedded graph that has all the information you need to make an educated investing decision. To see this code running. Go to the runtime link, click on the stocks_here, and click on Stocks. Here you will see the stocks widget that shows the AAPL stock history. You can also compare it to other companies like MSFT.

**Comment by noahahooja** [comment]
Nice api added and embedded pictures/videos about stocks, try to add parts where users can interact more with the idea of a stock simulator.

**Comment by ryanmgds** [comment]
Nice use of api, adds solid part to project allowing users to interact. Impressive work and i like the idea. Maybe now extend on with this into simulator.

**Comment by ghost** [comment]
Hey Brayden, from what I see you are working on the front end for your project in order to better show the theme. I like the chart you created because I think it is a great way to establish the theme. In order to take this to the next level maybe you can integrate a database/api into this in order to allow the user to have their own control over what they see.

**Comment by JettKim** [comment]
Dang Brayden you lookin good at coding. You should teach Nathan some of this impressive stuff.

**Comment by carterquartararo** [comment]
Good Idea to help add more of a body to the website and add more Depth. Good to see ideas like this as it is something that I  would of never thought of.
